workflow/validate/validate_test.go

```diff
@@ -302,6 +302,89 @@ func TestStepOutputReference(t *testing.T) {
 	assert.Nil(t, err)
 }
 
+
+var stepStatusReferences = `
+apiVersion: argoproj.io/v1alpha1
+kind: Workflow
+metadata:
+  generateName: step-output-ref-
+spec:
+  entrypoint: whalesay
+  templates:
+  - name: whalesay
+    inputs:
+      parameters:
+      - name: message
+        value: "value"
+    container:
+      image: docker/whalesay:latest
+    outputs:
+      parameters:
+      - name: outparam
+        valueFrom:
+          path: /etc/hosts
+  - name: stepref
+    steps:
+    - - name: one
+        template: whalesay
+    - - name: two
```

Comment:
Will step two actually run here?

Reply:
@Ark-kun This was just meant to test that the static validator accepted `"{{steps.one.status}}"` as a valid variable. But might as well make it a functional example, good catch.